wildfly-modules/jboss/container/wildfly/startup/artifacts/bin/openshift-launch.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+echo "`date "+%Y-%m-%d %H:%M:%S"` WildFly Image on OpenShift launch"
```

Comment:
What about : "Launching WildFly Server" instead of "WildFly Image on OpenShift launch"

Reply:
I change it, thank you for review :)